include/dxc/DXIL/DxilModule.h

```diff
@@ -381,24 +395,6 @@ class DxilModule {
   template<typename T> unsigned AddResource(std::vector<std::unique_ptr<T> > &Vec, std::unique_ptr<T> pRes);
   void LoadDxilSignature(const llvm::MDTuple *pSigTuple, DxilSignature &Sig, bool bInput);
 
-  // properties from HLModule preserved as ShaderFlags
-  bool m_bDisableOptimizations;
-  bool m_bUseMinPrecision;
-  bool m_bAllResourcesBound;
-  bool m_bResMayAlias;
-
-  // properties from HLModule that should not make it to the final DXIL
-  uint32_t m_IntermediateFlags;
-  uint32_t m_AutoBindingSpace;
-
-  // porperties infered from the DXILTypeSystem
-  bool m_bHasPayloadQualifiers;
```

Comment:
This is the variable that was removed because it was unused @pacxx? Do you remember why it was added in #3171 in the first place? It has never been used.